src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -352,18 +352,42 @@ public async Task Build_Invalid_Single_Params_File_ShouldFail_WithExpectedErrorM
         }
 
         [TestMethod]
-        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/.*/main\.bicepparam")]
+        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/Extends/main\.bicepparam")]
         [TestCategory(BaselineHelper.BaselineTestCategory)]
-        public async Task Build_params_to_stdout_with_non_bicep_references_should_succeed(EmbeddedFile paramFile)
+        public async Task Build_params_to_stdout_with_extends_should_succeed(EmbeddedFile paramFile)
         {
             var baselineFolder = BaselineFolder.BuildOutputFolder(TestContext, paramFile);
-            var outputFile = baselineFolder.GetFileOrEnsureCheckedIn("output.json");
+            var outputFile = baselineFolder.GetFileOrEnsureCheckedIn("main.json");
+
+            var clients = await MockRegistry.Build();
+            var settings = new InvocationSettings(new(TestContext, RegistryEnabled: true), clients.ContainerRegistry, clients.TemplateSpec);
+
+            var result = await Bicep(settings, "restore", baselineFolder.EntryFile.OutputFilePath);
+            result.Should().Succeed().And.NotHaveStdout().And.NotHaveStderr();
+
+            result = await Bicep(settings, "build-params", baselineFolder.EntryFile.OutputFilePath, "--stdout");
+            result.Should().Succeed();
+
+            var parametersStdout = result.Stdout.FromJson<BuildParamsStdout>();
+            // Force consistency for escaped newlines.
+            parametersStdout = parametersStdout with { templateJson = parametersStdout?.templateJson?.ReplaceLineEndings("\n") };
+            outputFile.WriteJsonToOutputFolder(parametersStdout);
+            outputFile.ShouldHaveExpectedJsonValue();
+        }
+
+        [TestMethod]
+        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/Extends/shared\.bicepparam")]
```

Comment:
Rather than creating a dedicated test for this, could you instead just create a file under `Files/BuildParamsCommandTests/UsingNone/main.bicep`?

Reply:
yes, I created a new set of files under `UsingNone` folder 👍